druid/examples/identity.rs

```diff
@@ -14,48 +14,34 @@
 
 //! An example of sending commands to specific widgets.
 //!
-//! This example is fairly contrived; the basic idea is that there is one
-//! "top widget", displaying random colors, and are three rows of widgets,
-//! each containing a ColorWell and two buttons.
-//! One button 'pins' the ColorWell, sending it a color to display.
-//! The second button 'syncs' the ColorWell, which makes it start displaying
-//! the same random colors as the top widget.
+//! This example is fairly contrived; the basic idea is that there are 2 counters,
+//! and 2 buttons. If you press button one, counter one goes up. If you press button
+//! two counter 2 goes up.
 //!
-//! The key insight is that each button is linked to a specific ColorWell, and
-//! can send messages that are only handled by that widget.
+//! The key insight is that each button is linked to a specific counter, and
+//! can send messages that are only handled by that widget/controler.
 //!
 //! This is a contrived example; if you were designing a real app you might
```

Comment:
```suggestion
//! If you were designing a real app you might
```

You already said this above.